release-notes/release-notes58.md

```diff
@@ -188,14 +188,21 @@ This section describes issues that users of {{site.productname}} 5.8 may encount
 
 **Outline**
 
-- [](#)
+- [Unexpected behavior when moving the caret using the PageDown key](#unexpectedbehaviorwhenmovingthecaretusingthepagedownkey)
+- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)
 
-###
+### Unexpected behavior when moving the caret using the PageDown key
 
-**Issue**: This issue affects ....
+**Issue**: This issue affects moving the caret over links using the PageDown key. If the caret is placed before a link and the PageDown key is pressed, the caret will move inside the link rather than past it.
 
 **Workaround**: There was no known workaround at the time of the release.
 
+### Accessibility when using keyboard navigation to resize the editor
+
+**Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA role to indicate the resize state for a screen reader or other assistive technology to consume.
```

Comment:
```suggestion
**Issue**: The editor can now be resized using keyboard navigation, however there is currently no available ARIA attributes to indicate the resize state for a screen reader or other assistive technology to consume.
```
This will entail more than just a role, so this just updates that to reflect that.